### DISCUSSION TITLE
Using sudo in Git Extensions

### DISCUSSION THREAD

**Discussion by klorinczi** [gitextensions/gitextensions#10036] [Q&A]
Hi,

I have Git Extensions v3.1.1 installed.

I would like to use sudo for all git commands within Git Extensions, because etckeeper needs root or sudo for /etc directory.

How can I add sudo before each git command execute?

**Comment by gerhardol** [comment]
You may implement a wrapper that has the same input and output as git.exe. Beware of quoting etc.
